portfolio/src/main/webapp/script.js

```diff
@@ -51,8 +52,26 @@ const handleClickProject = function() {
     });
 }
 
-function getRandomQuoteUsingArrowFunctions() {
-    fetch('/data').then(response => response.text()).then((quote) => {
-        document.getElementById('quote-container').innerHTML = quote;
+function getComments() {
```

Comment:
Functions earlier in this file use the form

`const ___ = function () {`

but these functions use

`function ____ () {`

could you pick one to be consistent with?